[FSTORE-588] Kafka Authoriser Update (#191)
logicalclocks/logicalclocks.github.io: docs/user_guides/projects/kafka/consume_messages.md

## Introduction

> A Consumer is a process which reads messages from a kafka topic. In Hopsworks, all user roles are capable of performing 'Read' and 'Describe' actions on Kafka topics within projects that he is a member of or are shared with him.

Reviewer: ```suggestion
A Consumer is a process which reads messages from a Kafka topic. In Hopsworks, all user roles are capable of performing 'Read' and 'Describe' actions on Kafka topics within projects that they are a member of or are shared with them.
```
